README.md

```diff
@@ -0,0 +1,112 @@
+![sttp-model](https://github.com/softwaremill/sttp-openai/raw/master/banner.png) Banner should be changed to sttp openAi
+
+[![Join the chat at https://gitter.im/softwaremill/sttp](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/softwaremill/sttp)
+[![CI](https://github.com/softwaremill/sttp-openai/workflows/CI/badge.svg)](https://github.com/softwaremill/sttp-openai/actions?query=workflow%3ACI+branch%3Amaster)
+
+[//]: # ([![Maven Central]&#40;https://maven-badges.herokuapp.com/maven-central/com.softwaremill.sttp&#41;&#40;https://maven-badges.herokuapp.com/maven-central/com.softwaremill.sttp&#41;)
+
+sttp is a family of Scala HTTP-related projects, and currently includes:
+
+* [sttp client](https://github.com/softwaremill/sttp): The Scala HTTP client you always wanted!
+* [sttp tapir](https://github.com/softwaremill/tapir): Typed API descRiptions
+* sttp openai: this project. Scala client wrapper for OpenAi API. Use the power of ChatGPT inside your code!
+
+## Quickstart with sbt
+
+Add the following dependency:
+
+[//]: # (```scala)
+[//]: # ("com.softwaremill.sttp.openai" %% "core" % "?.?.?")
+[//]: # (```)
+
+sttp openai is available for Scala 2.12 and Scala 3.2.2
+
+## Project content
+
+Available endpoints with modeled response classes include:
```

Comment:
Hm maybe it would be easier to point to OpenAI docs (to a specific version, if they are versioned) and list the endpoints which are not implemented?